src/help/validations.csv

```diff
@@ -52,33 +53,33 @@ C,C4,"ObligationsDeliveredOrdersUnpaidTotal in File C = USSGL 4901 + 4981 in Fil
-C/D1/D2,C24,"If AllocationTransferAgencyIdentifier in File C is populated and valid, then it is acceptable that the associated Award ID does NOT exist in File D1 or D2.",n/a,Implemented,Linkage
+"C, D1, D2",C23,"For each unique award ID (PIID or combination of PIID/ParentAwardId for procurement; FAIN and/or URI for financial assistance) in File C, the sum of each TransactionObligatedAmount should match (but with opposite signs) the sum of the FederalActionObligation (in either D1 or D2) or OriginalLoanSubsidyCost (only in D2) amounts reported in D1 or D2.  For example, if the TransactionObligatedAmounts for a FAIN add to -100 in File C, the sum of the FederalActionObligation for that FAIN in D2 should be 100.",Warning,Implemented,Calculation
+"C, D1, D2",C24,"If AllocationTransferAgencyIdentifier in File C is populated and valid, then it is acceptable that the associated Award ID does NOT exist in File D1 or D2. ",n/a,Implemented,Linkage
 D2,D1,"Element value must be a valid enumeration as described in the ""Domain_Values.xlsx"" spreadsheet.",n/a,Implemented in ASP,Requirement
 D2,D2,Must be blank for aggregate records; required for non-aggregate records.,n/a,Implemented in ASP,Element Conditionally Required
 D2,D3,"Must be a valid date with a value between 19991001 and 20991231. Date must follow YYYYMMDD format, where YYYY is the year, MM the month and DD the day. A future ActionDate is valid only if it occurs within the current fiscal year.",n/a,Implemented in ASP,Element Conditionally Required
 D2,D4,PeriodOfPerformanceCurrentEndDate >= PeriodOfPerformanceStartDate,n/a,Implemented in ASP,Requirement
 D2,D5,"Either FAIN or URI (Unique Record Identifier) is required for corrections and deletions (i.e., when CorrectionLateDeleteIndicator = C or D). The combination of FAIN and AwardModificationAmendmentNumber must be unique if the URI is not provided. The URI must be unique if the FAIN and AwardModificationAmendmentNumber are not provided.",n/a,Implemented in ASP,Requirement
 D2,D6,"The combination of FAIN, AwardModificationAmendmentNumber, URI, and AwardingSubTierAgencyCode must be unique unless the CorrectionLateDeleteIndicator = C or D.",n/a,Implemented in ASP,Element Conditionally Required
-D2,D7,"This is required only when the combination of FAIN and AwardModificationAmendmentNumber is not unique. Field must contain a unique value except when the CorrectionLateDeleteIndicator contains the code ""C"" or ""D"". This field is required for aggregate records.",n/a,Implemented in ASP,Element Conditionally Required
+D2,D7,This is required only when the combination of FAIN and AwardModificationAmendmentNumber is not unique. Field must contain a unique value except when the CorrectionLateDeleteIndicator contains the code “C” or “D”. This field is required for aggregate records. ,n/a,Implemented in ASP,Element Conditionally Required

```

Comment:
Didn't we have problems last time we changed quotes to smart quotes or am I misremembering? Also, why did we add a space here? There are extra spaces in other changes as well, not going to comment on all of them in case it doesn't matter.